ext/phar/tests/phar_setsignaturealgo1.phpt

```diff
@@ -2,7 +2,6 @@
 Phar::setSignatureAlgorithm()
 --SKIPIF--
 <?php if (!extension_loaded("phar")) die("skip"); ?>
-<?php if ( extension_loaded("hash")) die("skip extension hash conflicts"); ?>
```

Comment:
Possibly same as ext/phar/tests/phar_setsignaturealgo2.phpt now?

Reply:
This test is actually part of some forgotten functionality of phar. Phar::GPG. And is currently never executed actually. So, yes basically the same as phar_setsignaturealgo2. Should we remove the unfinished phar_setsignaturealgo1.phpt file?

Reply:
Yes, if `Phar::PGP` doesn't exist it makes no sense to keep the test...